refinery/tool_manager/models.py

```diff
@@ -31,22 +33,28 @@ class Parameter(models.Model):
     uuid = UUIDField(unique=True, auto=True)
     name = models.TextField(max_length=100)
     description = models.TextField(max_length=500)
-    is_user_adjustable = models.BooleanField(default=False)
+    is_user_adjustable = models.BooleanField(default=True)
     value_type = models.CharField(choices=VALUE_TYPES, max_length=25)
     default_value = models.TextField(max_length=100)
 
     def __str__(self):
         return "{}: {} - {}".format(self.value_type, self.name, self.uuid)
 
+    def get_galaxy_workflow_step(self):
+        try:
+            return GalaxyParameter.objects.get(
+                uuid=self.uuid
+            ).galaxy_workflow_step
+        except (GalaxyParameter.DoesNotExist,
+                GalaxyParameter.MultipleObjectsReturned):
+            return None
```

Comment:
Yes, this just allows the Parameter Serializer to include the GalaxyParameter models fields when need be, and disclude them from the response if we are dealing with parent Parameter objects